rpm/del-fixtures-sign.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+#
+# Delete signatures of all packages recursively in target directory.
+# One RPM utility `rpmsign` must be installed.
+
+set -euo pipefail
+
+# Print a message to stdout explaining how to use this script.
+#
+# NOTE: $0 corresponds to the the script's name only in some shells and in some
+# contexts.
+show_help() {
+fmt <<EOF
+usage: del-fixtures-sign.sh <target_dir>
+
+Delete signatures of all RPM/SRPM/DRPM files recursively in <target_dir>
+
+EOF
+cat <<EOF
+<target_dir>
+    The directory from which package's signature will be removed.
+EOF
+}
+
+# Fetch target_dir from user.
+if [ "$#" -lt 1 ]; then
+    echo 1>&2 'Error: Too few arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+elif [ "$#" -gt 1 ]; then
+    echo 1>&2 'Error: Too many arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+else
+    target_dir="$(realpath --canonicalize-missing "${1}")"
```

Comment:
`--canonicalize-missing` is an inappropriate option to use here. From `realpath`'s man page:

```
       Print the resolved absolute file name; all but the last component must exist

       -e, --canonicalize-existing
              all components of the path must exist

       -m, --canonicalize-missing
              no path components need exist or be a directory
```

`--canonicalize-existing` should be used.